From 5d2c0848122678c3cd75815c2c4808bca701b6cd Mon Sep 17 00:00:00 2001
From: Guilhem Saurel <guilhem.saurel@laas.fr>
Date: Tue, 2 Mar 2021 15:05:12 +0100
Subject: [PATCH 2/3] Disable BUILD_TESTING for ROS

As this would add an extra dependency to example-robot-data
---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d0b167a..81dc0c4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -38,6 +38,7 @@ CHECK_MINIMAL_CXX_STANDARD(11 ENFORCE)
 ADD_PROJECT_DEPENDENCY(dynamic-graph REQUIRED PKG_CONFIG_REQUIRES dynamic-graph)
 ADD_PROJECT_DEPENDENCY(pinocchio REQUIRED PKG_CONFIG_REQUIRES pinocchio)
 ADD_PROJECT_DEPENDENCY(Boost REQUIRED COMPONENTS regex)
+SET(BUILD_TESTING OFF CACHE BOOL "")
 IF(BUILD_TESTING)
   ADD_PROJECT_DEPENDENCY(example-robot-data)
   FIND_PACKAGE(Boost REQUIRED COMPONENTS unit_test_framework program_options)
-- 
2.30.2

